react-native/react/menubar/index.render.desktop.js

```diff
@@ -2,26 +2,65 @@
 /*eslint-disable react/prop-types */ // Since we're using flow types for props
 
 import React, {Component} from '../base-react'
-import {FlatButton} from 'material-ui'
 import resolveAssets from '../../../desktop/resolve-assets'
 
 import {intersperseFn} from '../util/arrays'
 import {parseFolderNameToUsers, canonicalizeUsernames, stripPublicTag} from '../util/kbfs'
 
-import commonStyles, {colors} from '../styles/common'
+import {globalStyles, globalColors} from '../styles/style-guide'
+import Text from '../common-adapters/text'
+import Button from '../common-adapters/button'
+import Divider from '../common-adapters/divider'
 
 // This is the only data that the renderer cares about for a folder
 import type {FolderInfo} from './index.render'
 
-const folderIcon = {
-  public: {
-    empty: `file:///${resolveAssets('../react-native/react/images/folders/kb-folder-public-empty.svg')}`,
-    full: `file:///${resolveAssets('../react-native/react/images/folders/kb-folder-public-full.svg')}`
-  },
-  private: {
-    empty: `file:///${resolveAssets('../react-native/react/images/folders/kb-folder-private-empty.svg')}`,
-    full: `file:///${resolveAssets('../react-native/react/images/folders/kb-folder-private-full.svg')}`
-  }
+function iconPath (isPublic, isEmpty) {
+  const pubPart = isPublic ? 'public' : 'private'
+  const emptyPart = isEmpty ? 'empty' : 'full'
+  return 'file:///' + resolveAssets(`../react-native/react/images/folders/kb-folder-${pubPart}-${emptyPart}.svg`)
+}
+
+const Header = props => {
```

Comment:
Using the stateless components model instead of entire classes for little helpers